contribute/work-in-progress.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: 'In Progress Work in Beam'
+permalink: /contribute/work-in-progress/
+---
+
+# Work In Progress in the Apache Beam Project
+
+As mentioned in the [Contribution Guide](/contribute/contribution-guide/), all work in Beam is tracked in our [Apache JIRA](https://issues.apache.org/jira/browse/BEAM). In addition, the following types of work may be of particular interest to the Beam community.
+
+* TOC
+{:toc}
+
+
+## Starter Tasks
+
+The community regular tags good getting started tasks with the label `newbie` or `starter`. Use a quick [JIRA search](https://issues.apache.org/jira/issues?jql=project%20%3D%20BEAM%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20in%20(newbie%2C%20starter)) to identify ways you can get started [contributing](/contribute/contribution-guide/) to Beam.
```

Comment:
'starter' wins as it's much more heavily used and also what Flink and Spark use.